modules/admin_manual/pages/release_notes.adoc

```diff
@@ -21,6 +24,45 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.2.1
+
+ownCloud Server 10.2.1 is a bug fix and maintenance release taking care of several bugs and known issues. 
+Please find, below, the changes in ownCloud Server 10.2.1 that need your attention. 
+You can also read {owncloud-server-changelog-url}[the full ownCloud Server changelog] for further details on what has changed.
+It is recommended to schedule an upgrade to this version soon, especially if you're running 10.2.0 already.
+
+TIP: No occ upgrade is required when upgrading from 10.2.0.
+
+=== Improved Performance For Storage Encryption With Master Key
+
+ownCloud Server offers two ways for key management with storage encryption. 
+Either a central master key pair or individual user key pairs are used to encrypt/decrypt data. 
+Previously both modes used the same mechanisms which resulted in potentially significant overhead when master key encryption was used as user key encryption relies on so-called `share keys` which are necessary to allow share recipients to decrypt shared files.
+
+With master key encryption, `share keys` are redundant as you have one central key that can be used to decrypt all files. 
+Version 10.2.1 corrects this behavior by dropping `share keys` for master key encryption, thereby increasing the performance dramatically, especially when sharing folders with many files as said keys do not have to be generated anymore for each file.
+
+=== Solved Issues
+
+* **Fixed reshare permission issue** +
+ownCloud Server 10.2.0 introduced an issue that allows users to increase sharing permissions beyond their own permissions in a reshare scenario: When user A shares a folder "_Project_" with user B, granting only read and share permission, then the Sharing API allowed user B to reshare a subfolder of "_Project_" with user C granting full permissions or to create a public link on the shared folder, respectively. 
```

Comment:
@settermjd @pmaier1 This has not been introduced in 10.2.0. It has been undiscovered for a long time.

Reply:
Thx, changed.